src/App.js

```diff
@@ -1,16 +1,55 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList'
+
+const myData=[
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state={
+      data:myData,
+       item:'',
+    };
+  }
+
+  addItem = items =>{
+    items.preventDefault();
+    const newItems={task:this.state.item, id:Date.now()};
+    this.setState({
+      data:[...this.state.data, newItems],
+      item:'',
+    });
+  };
+
+  updateItem=items=>{
+    this.setState({
+      [items.target.name]:items.target.value}
+      );
```

Comment:
Since the `items` parameter here seems to refer to an event being processed by a listener, I would advise renaming it to something more declarative like `event` or even just `e` (which is quite a common convention anyway, despite the single-letter name).